.circleci/config.yml

```diff
@@ -98,19 +120,32 @@ workflows:
       - build:
           requires:
             - linter
-          filters:
-            branches:
-              only: master
+
       - docs:
           requires:
             - linter
           filters:
             branches:
-              only: master
+              only: 
+                - master
+
+      - unit_tests:
+          requires:
+            - build
+
       - release:
           requires:
             - build
             - docs
           filters:
             branches:
               only: master
+              
+
+      - release_widgets:
+          requires:
+            - build
+            - unit_tests
+          filters:
+            branches:
+              only: feat/cc-widgets
```

Comment:
This make sure we only release ccwidgets when we push onto feat/cc-widgets branch